clodius/multivec.py

```diff
@@ -41,27 +44,31 @@ def bedfile_to_multivec(input_filenames, f_out,
     print('base_resolution:', base_resolution)
     warned = False
 
-    for lines in zip(*files):
+    for count, lines in enumerate(zip(*files)):
         # Identifies bedfile headers and ignore them
-        if "browser" == lines[0].decode('utf8')[0:7] or "track" in lines[0].decode('utf8')[0:6]:
+        if "browser" == lines[0][0:7] or "track" in lines[0][0:6]:
             continue
 
         chrom, start, end, vector = bedline_to_chrom_start_end_vector(
             lines, row_infos)
         # if vector[0] > 0 or vector[1] > 0:
-        # print("c,s,e,v", chrom, start, end, vector)
+
+        if len(vector) < len(lines) * num_rows:
+            logger.warn('Lines contain less columns than expected: %s', lines)
+            vector += [np.nan] * (len(lines) * num_rows - len(vector))
 
         if end % base_resolution != 0 or start % base_resolution != 0 and not warned:
```

Comment:
Not in this commit, but the order of operations for `or`/`and` is like `+`/`*`: Should this be
```suggestion
         if (end % base_resolution != 0 or start % base_resolution != 0) and not warned:
```
?